add-ons/dspace-docker-tools/ingestAIP.sh

```diff
@@ -2,6 +2,8 @@
 # Once this file has been saved to a docker volume, the ingest step will not be re-run
 CHECKFILE=/dspace/assetstore/ingest.hasrun.flag
 
+env | egrep "__.*=" | egrep -v "=.*__" | sed -e s/__/./g > /dspace/config/environment.cfg
```

Comment:
This pipe excludes any variable whose *value* contains a double underscore.  That's highly unlikely to cause problems, but I did work out a different one that doesn't have that issue:
``env | awk -F = -e '/__.*=/ { name = $1; divide = index($0, "="); value = substr($0, divide+1); gsub(/__/, ".", name); print name "=" value }' > /dspace/config/environment.cfg``